cmd/dep/main.go

```diff
@@ -153,6 +166,26 @@ func (c *Config) Run() int {
 		return successExitCode
 	}
 
+	// 'dep help constraints' generates documentation on the exact syntax for version constraints
```

Comment:
This is more of printing. The block above this is actual generation of a .go file. Let's change this to
`prints version constraints syntax documentation.`, ending with a period 🙂